homeassistant/components/rfxtrx/sensor.py

```diff
@@ -92,6 +92,8 @@ def sensor_update(event):
                     continue
                 sensor = sensors[data_type]
                 sensor.event = event
```

Comment:
Let's put all this logic in a `@callback` method on the entity too. We're moving away from having schedule_state/write_state being called from outside of the entity

Reply:
Yup done in follow-up. Whole unit logic is getting a rewrite